packages/commonwealth/client/scripts/views/components/component_kit/cw_empty_state.tsx

```diff
@@ -2,23 +2,30 @@ import React from 'react';
 
 import 'components/component_kit/cw_empty_state.scss';
 
+import { AutomationTestProps } from '../../pages/error';
 import { CWIcon } from './cw_icons/cw_icon';
 import type { IconName } from './cw_icons/cw_icon_lookup';
 import { CWText } from './cw_text';
 
 type EmptyStateProps = {
   content: string | React.ReactNode;
   iconName?: IconName;
-};
-
-export const CWEmptyState = (props: EmptyStateProps) => {
-  const { content, iconName } = props;
+} & AutomationTestProps;
 
+export const CWEmptyState = ({
+  content,
+  iconName,
+  ...rest
```

Comment:
can't we just use `testId` instead of ...rest ?

Reply:
we can actually, i wanted to use the exact param `data-testid` so had to destructure like that. What do you prefer?

Reply:
I see. Current syntax is a bit clunky though so I think `testId` would be fine